### ISSUE TITLE
FirebasePlugin.verifyPhoneNumber() not working in iOS device, but works on Android

### ISSUE THREAD

**Issue by ganiru** [dpa99c/cordova-plugin-firebase#36]
**Describe the bug**
Trying to do phone authentication  on an Ionic 3 app. The function `window.FirebasePlugin.verifyPhoneNumber()` is completely skipped in iOS (no error messages, nada), but works fine in Android.

**To Reproduce**
1. (Testing on iPhone 6) Installed the plugin `ionic cordova plugins add cordova-plugin-firebasex`
2. In the code (.ts file), declare var window
3. call the function `window.FirebasePlugin.verifyPhoneNumber(phonenumber, 60, (credential)=> {console.log("credential:", credential})`
4. console.log never gets called

**Expected behavior**
The credential should be logged

**Screenshots**
None

**Console Logs**
No logs are shown

**Plugin Version**
version 3.0.8

**Desktop (please complete the following information):**
 - OS: Developing on MacOS High Sierra

**Smartphone (please complete the following information):**
 - Device: iPhone6
 - OS: iOS 12.3.1
 - Browser: safari

**Additional context**
It's an Ionic 3 app. This works fine on Android.

**Comment by dpa99c** [comment]
This functionality has been directly carried over from [cordova-plugin-firebase](https://github.com/arnesson/cordova-plugin-firebase) so I presume it's related to https://github.com/arnesson/cordova-plugin-firebase/issues/1050 and https://github.com/arnesson/cordova-plugin-firebase/issues/965.

I'm working my way through the code of this plugin and rewriting it so will get to testing and fixing this at some point soon.

**Comment by ganiru** [comment]
Thanks for the feedback. I was able to get it to work thanks to the link you provided [https://github.com/arnesson/cordova-plugin-firebase/issues/1050](url). It turns out I had to enable **Remote Notifications** in XCode, and also added the **custom URL** in the Info tab. I just wish it had at least generated an error to let the developer know what the issue is, rather than just be silent.

Thanks for your hard work on this plugin. Keep up the good work!
